### ISSUE TITLE
FPS Problem

### ISSUE THREAD

**Issue by Dylan404** [Haleth/Aurora#157]
From the version 8.1.0.2 Alpha or Beta I found out with the addon (Addon Usage) that Aurora uses up to 70% of the CPU ingame and negatively influences the FPS.
Please fix this problem.

**Comment by CoreVale** [comment]
I have been experiencing this as well, memory use will just climb. literally standing still doing nothing, watching the memory spike and fps drop. Had to disable until bug is fixed

**Comment by Dylan404** [comment]
Use currently the version 8.1.0.2.
I have no problems with the version.

**Comment by Dylan404** [comment]
I've tested it ... what can I say better, but still too much (50%) and LFGFrame.lua error.

**Comment by Gethe** [comment]
How are you measuring usage? Is it steadily increasing in usage?

What is the difference in framerate vs when Aurora is not enabled?

**Comment by Dylan404** [comment]
via Addon Usage can read how the CPU cores are used in WoW and how much RAM is used. No matter what I do the CPU usage is <50% with version 8.1.0.2 only> 10%. The FPS drop by 5-10 FPS and it jerks. Many of my guildmates have the same problem and have switched back to the old version.

**Comment by Dylan404** [comment]
https://wow.curseforge.com/projects/addon-usage

**Comment by Dylan404** [comment]
I made a screenshot for comparison.

Aurora 1.8.0.2 : [https://imgur.com/a/5YWMEMH](url)
Aurora 1.8.0.4 : [https://imgur.com/a/xt35MsD](url)

**Comment by Gethe** [comment]
Is that using the latest alpha? If you could, please try that one and do `/aurora profile`

**Comment by Dylan404** [comment]
still the same problem

**Comment by Gethe** [comment]
Please try the slash command I mentioned, and post a screenshot with it and addon usage.

**Comment by Dylan404** [comment]
Give me a second
[https://imgur.com/a/13hcpBb](url)

**Comment by Gethe** [comment]
You should have got a frame like this with that slash command. ![this](https://i.imgur.com/gIvyDCV.png):


Make sure you're using the alpha version on curse or download it directly from here.

**Comment by weiz** [comment]
I also have had an _extremely_ high CPU usage, and even stopped using it around Fall 2018 because it was just crazy high. I just got 8.1.5.0 today after seeing the notes about the performance changes so I gave it a try: As well I went ahead and ran that profiling command for you (in case it helps)


Here are two screenshots while being afk for the most part in Dalaran.
 ![profiling](https://i.imgur.com/sJNKrD9.png):
 ![profiling](https://i.imgur.com/8U74c10.png):

If you want me to test anything, or dump info/data for you; just let me know as I would love for this to be working properly on my end!

**Comment by Gethe** [comment]
Is that on a fresh login or after playing for a while? If the later, what were you doing?

**Comment by weiz** [comment]
I don't think it was right at login, but I could run it right after login if that would help. 
I wasn't  doing a whole lot game-wise at the time. Mostly just afk while looking into the patch notes for 8.1.5, etc.

If there are a few specific situations you would like me to test for, just let me know what to do, when to do it, and when to get screenshots. I can do a few different situations if that helps, (such as with or without addons or specific ones) just let me know what you want.

**Comment by Gethe** [comment]
try running it with just Aurora

**Comment by weiz** [comment]
You got it: 
![only aurora](https://i.imgur.com/htv665e.png)
This screenshot was taken after logging with just Aurora.
Spawned in org and only used guild chat. I don't think I even opened up any UI windows or anything else during this. 

If you want any other situations tested just let me know and I'll get to them when I can.
Hope this helps!
